uiowa/uiowa
# Only change image size when previously set to small

Resolves #4441 





# How to test


`blt ds --site=sandbox.uiowa.edu`
Try editing
https://sandbox.prod.drupal.uiowa.edu/node/616/layout
When configuring the video, the image size should remain set to "Medium"
Try different combinations of saving the block as medium or large, and see that it saves correctly and opens to the same size when re-editing
Try removing the video or adding a new block, setting the image size to small, and see that it still updates to Large when then adding a remote video
